src/cohere/manually_maintained/cache.py

```diff
@@ -0,0 +1,23 @@
+import typing
+import time
+
+
+class CacheMixin:
+    # A simple in-memory cache with TTL (thread safe). This is used to cache tokenizers at the moment.
+    _cache: typing.Dict[str, typing.Tuple[typing.Optional[float], typing.Any]] = dict()
```

Comment:
does it persist accross executions?

Reply:
no. it's per client (so if you define multiple clients in one runtime, each will contain its own cache)

Reply:
ook, i'm thinking this might be slow for users that are running a script many times?

Reply:
if they re init the client, yes. 
I was already looking into adding a persistent FS cache, but I thought that can come as an enhancement?

Reply:
yeah sg